api-reference/20 Data Visualization Widgets/dxChart/1 Configuration/argumentAxis/categories.md

```diff
@@ -57,6 +57,55 @@ To sort the arguments, for example, alphabetically, you need to assign an array
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <DxChart ... >
+            <DxArgumentAxis 
+                :categories="['Africa', 'Antarctica', 'Asia', 'Australia',
```

Comment:
Let's declare this array as a component property in Angular, Vue, and React to make the samples more readable and avoid re-rendering in React.